sql_extensions/migrations/00002_latest_periodic_state.sql

```diff
@@ -0,0 +1,227 @@
+    (inbox_latest_periodic_state.data -> 'market_metadata' ->> 'market_id')::NUMERIC = (EXCLUDED.data -> 'market_metadata' ->> 'market_id')::NUMERIC
+  AND
+    (inbox_latest_periodic_state.data -> 'periodic_state_metadata' ->> 'emit_market_nonce')::NUMERIC < (EXCLUDED.data -> 'periodic_state_metadata' ->> 'emit_market_nonce')::NUMERIC
+  AND
+    NEW.data -> 'periodic_state_metadata' ->> 'period' = '86400000000';
+  RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE TRIGGER update_latest_periodic_state
+AFTER INSERT ON inbox_events
+FOR EACH ROW
+WHEN (new.event_name = 'emojicoin_dot_fun::PeriodicState')
+EXECUTE PROCEDURE UPDATE_LATEST_PERIODIC_STATE();
+
+DROP VIEW market_data;
+-- noqa: disable=ST06
+CREATE VIEW market_data AS
+SELECT
+    -- General data
+    (registration.data -> 'market_metadata' ->> 'market_id')::NUMERIC AS market_id,
+    (registration.data -> 'market_metadata' ->> 'emoji_bytes') AS emoji_bytes,
+    (registration.data -> 'market_metadata' ->> 'market_address') AS market_address,
+    -- Latest state data
+    state.transaction_version,
+    (state.data -> 'instantaneous_stats' ->> 'market_cap')::NUMERIC AS market_cap,
+    (state.data -> 'state_metadata' ->> 'bump_time')::NUMERIC AS bump_time,
+    (state.data -> 'cumulative_stats' ->> 'n_swaps')::NUMERIC AS n_swaps,
+    (state.data -> 'cumulative_stats' ->> 'n_chat_messages')::NUMERIC AS n_chat_messages,
+    (state.data -> 'last_swap' ->> 'avg_execution_price_q64')::NUMERIC AS avg_execution_price_q64,
+    (state.data ->> 'lp_coin_supply')::NUMERIC AS lp_coin_supply,
+    (state.data -> 'clamm_virtual_reserves' ->> 'base')::NUMERIC AS clamm_virtual_reserves_base,
+    (state.data -> 'clamm_virtual_reserves' ->> 'quote')::NUMERIC AS clamm_virtual_reserves_quote,
+    (state.data -> 'cpamm_real_reserves' ->> 'base')::NUMERIC AS cpamm_real_reserves_base,
+    (state.data -> 'cpamm_real_reserves' ->> 'quote')::NUMERIC AS cpamm_real_reserves_quote,
+    -- Volume data
+    COALESCE(volume.all_time_volume, 0) AS all_time_volume,
+    DAILY_VOLUME(volume) AS daily_volume, -- noqa: RF02
+    (periodic_state.data ->> 'tvl_per_lp_coin_growth_q64')::NUMERIC AS tvl_per_lp_coin_growth_q64
+FROM (
+    SELECT data FROM inbox_events WHERE event_name = 'emojicoin_dot_fun::MarketRegistration'
+) AS registration
+LEFT JOIN inbox_latest_state AS state
+    ON (registration.data -> 'market_metadata' ->> 'market_id')::NUMERIC = state.market_id
+LEFT JOIN inbox_volume AS volume
+    ON (registration.data -> 'market_metadata' ->> 'market_id')::NUMERIC = volume.market_id
+LEFT JOIN inbox_latest_periodic_state AS periodic_state
+    ON (registration.data -> 'market_metadata' ->> 'market_id')::NUMERIC = periodic_state.market_id;
+-- noqa: disable=ST06
+
+CREATE INDEX inbox_latest_periodic_state_by_tvl ON inbox_events (
+    ((data ->> 'tvl_per_lp_coin_growth_q64')::NUMERIC) DESC
+) WHERE (data ->> 'ends_in_bonding_curve')::BOOLEAN = true;
+
+-- }}}
+
+-- ADD LOCK TO PREVIOUS TRIGGERS {{{
```

Comment:
Is there anything from the last migration that is modified here aside from adding the lock?

Reply:
Confirming from eng call that nothing has been changed